ci-operator/config/openshift/svt/openshift-svt-master__regression-aws-4.17-nightly-x86.yaml

```diff
@@ -0,0 +1,37 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: regression-loaded-up
```

Comment:
The naming can be more meaningful to match the test case/script like `regression-scalability-loaded-projects`